libjwt/jwt.c

```diff
@@ -235,6 +235,18 @@ static long get_js_int(json_t *js, const char *key)
 	return val;
 }
 
+static int get_js_bool(json_t *js, const char *key)
+{
+	int val = -1;
+	json_t *js_val;
+
+	js_val = json_object_get(js, key);
+	if (js_val)
+		val = (int)json_boolean_value(js_val);
```

Comment:
You're going to need to add something to define `json_boolean_value` for older JANSSON.